apache-spark-worker/build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+RUN apt-get update && \
+  apt-get install -y \
+    python \
+    libnss3 \
+    openjdk-8-jdk \
+    curl
+
+COPY download.sh /home/
+RUN sh /home/download.sh
+
+WORKDIR /opt/apache-spark
+RUN sed -i "s/<joda.version>2.5<\/joda.version>/<joda.version>2.9.4<\/joda.version>/" pom.xml
```

Comment:
This is a bit wonky. The `.` has special meaning in regular expressions and should be escaped. More importantly, sed doesn't fail if no substitution is made. I think it would be better to `diff -u` the before and after versions of pom.xml, store `diff`'s output into a `.patch` file, copy the patch file into the container and then invoke `patch` inside the container. This is a good description:

http://www.thegeekstuff.com/2014/12/patch-command-examples